### PR TITLE
Add show modified date to load menu

### PR DESCRIPTION
**Pull request by jineyne**
If you don't mark the date of modification, it can cause discomfort to the player.

It's not my story. the real thing

### CODE REVIEW

File: app/src/main/java/com/qsp/player/game/MainActivity.java [hunk 1]
```diff
@@ -484,9 +486,14 @@ private void addSaveSlotsSubMenu(MenuItem parent, final SlotAction action) {
...
-            String title = String.format(
-                    getString(file != null ? R.string.slotPresent : R.string.slotEmpty),
-                    i + 1);
+            String title = null;
+
+            if (file != null) {
+                final String lastMod = (String) DateFormat.format("yyyy-MM-dd kk:mm:ss", file.lastModified());
+                title = String.format(getString(R.string.slotPresent), i + 1, lastMod);
+            } else {
+                title = String.format("Slot %d - [Empty]", i + 1);
```
Comment (seedhartha) [inline app/src/main/java/com/qsp/player/game/MainActivity.java]: Should be `getString(R.string.slotEmpty, i + 1)` to support localization.

File: app/src/main/java/com/qsp/player/game/MainActivity.java [hunk 2]
```diff
@@ -484,9 +486,14 @@ private void addSaveSlotsSubMenu(MenuItem parent, final SlotAction action) {
...
             final String filename = getSaveSlotFilename(i);
             final DocumentFile file = savesDir.findFile(filename);
-            String title = String.format(
-                    getString(file != null ? R.string.slotPresent : R.string.slotEmpty),
-                    i + 1);
+            String title = null;
+
+            if (file != null) {
+                final String lastMod = (String) DateFormat.format("yyyy-MM-dd kk:mm:ss", file.lastModified());
+                title = String.format(getString(R.string.slotPresent), i + 1, lastMod);
```
Comment (seedhartha) [inline app/src/main/java/com/qsp/player/game/MainActivity.java]: Can be simplified to `getString(R.string.slotPresent, i + 1, lastMod)`. See https://developer.android.com/reference/android/content/Context#getString(int,%20java.lang.Object...)

File: app/src/main/java/com/qsp/player/game/MainActivity.java [hunk 3]
```diff
@@ -484,9 +486,14 @@ private void addSaveSlotsSubMenu(MenuItem parent, final SlotAction action) {
         for (int i = 0; i < MAX_SAVE_SLOTS; ++i) {
             final String filename = getSaveSlotFilename(i);
             final DocumentFile file = savesDir.findFile(filename);
-            String title = String.format(
-                    getString(file != null ? R.string.slotPresent : R.string.slotEmpty),
-                    i + 1);
+            String title = null;
+
+            if (file != null) {
+                final String lastMod = (String) DateFormat.format("yyyy-MM-dd kk:mm:ss", file.lastModified());
```
Comment (seedhartha) [inline app/src/main/java/com/qsp/player/game/MainActivity.java]: `DateFormat.format` returns a `CharSequence` object, which is not guaranteed to be a `String`. It works, but `DateFormat.format("yyyy-MM-dd kk:mm:ss", file.lastModified()).toString()` would be more robust.

**Comment by jineyne** [comment]
Sure thing! all fixed

**Comment by seedhartha** [comment]
Merged as e140e9f50a7d076c3092abbbe933f662181d27f3, thanks!

**Pull request closed without merging**